wellcomeml/ml/bert_classifier.py

```diff
@@ -188,5 +188,5 @@ def save(self, model_path):
 
     def load(self, model_path):
         """Loads model from directory model_path. Tokenizer initialised from param pretrained"""
-        self.init_model()
+        self._init_model()
```

Comment:
this is a bit duplicated because in _init_model() you also do `TFBertForSequenceClassification.from_pretrained()`?

Reply:
if the loading takes a long time then I guess you might just want to take parts of _init_model() rather than running it all?